### DISCUSSION TITLE
zweiter Heizkreis

### DISCUSSION THREAD

**Discussion by kaiherzig** [xgcssch/SystaComfortPrometheusExporter#9] [Q&A] [answered]
Hallo Sönke,

wo müsste dich denn drehen, damit ich noch einen zweiten Heizkreis auslesen kann?

Viele Grüße
Kai

**Comment by xgcssch** [comment, marked as answer]
Hallo Kai!
Entschuldige bitte die späte Rückmeldung.
Ich denke, dass die Anlage die Daten in einem Paket mitschickt. Ich würde folgendermaßen vorgehen:
1. wie in [den FAQs bei der Fehlersuche (am Ende)](https://github.com/xgcssch/SystaComfortPrometheusExporter/blob/main/README_de.md#fehlersuche) beschrieben, den Container mit einem Verbosity Level von 5 laufen lassen. In der Ausgabe werden dann alle Werte aller Pakettypen ausgegeben.
2. Parallel dazu das Monitoring im Serviceprogramm oder die S-Touch Anwendung starten. Dort einen charakteristischen Wert des zweiten Heizkreises raussuchen. Bspw. Vorlauf- oder Rücklauftemperatur. Ausgehend davon, dann versuchen diesen Wert in den ausgegebenen Daten zu finden.
3. Wenn du die Daten entsprechend gefunden hast, gibt es zwei Möglichkeiten: a) du machst einen Issue auf, wo zu mir reinschreibst bei welchem Pakettyp und welchem Index sich die einzutragenden Werte befinden. Dann werde ich das entsprechend einpflegen und eine neue Version bereitstellen, oder du änderst und bildest das Programm selber und machst einen Pull Request, welchen ich dann übernehme.
In Ermangelung eines zweiten Heizkreises kann ich das leider nicht alleine machen.

Lieben Gruß
Sönke

**Reply by kaiherzig** [reply]
Hi Sönke,
4 ist "Puffertemperatur oben TPO"
5 ist "Puffertemp Unten TPU"
7 ist Vorlauftemp. Kreis 2
8 ist Rücklauftemp. Kreis 2
24 & 25 (Werte sind identisch und laufen identisch) sollten "Soll Vorlauf HK1" sein
26 sollte "Soll RaumTemp HK2" sein
27 & 28 (Werte sind identisch und laufen identisch) sollten "Soll Vorlauf HK2" sein.
33 "Sollwert Puffer"
70 "Sollwert Speicher Temp"
78 "HK2 RaumTemp Normal"
79 "HK2 RaumTemp Komfort"
80 "HK2 RaumTemp Absenken"
182 "solare leistung"
183 sollte Tagesgewinn Solare leistung 
156 sollte Max. Kollektor Temp sein
184 sollte Solar Gesamtgewinn in kWh sein

**Reply by xgcssch** [reply]
Habe einen Issue #10 dafür aufgemacht. Sobald ich Zeit habe, ergänze ich da.
Gruß
Sönke

**Comment by kaiherzig** [comment]
Super, vielen Dank!
muss ich jetzt das JSON aus Grafana ebenfalls updaten?

**Reply by xgcssch** [reply]
Ich habe an der Grafanakonfiguration keine Änderung gemacht, da bei mir keine Daten kommen.
Ev. hast du ja Lust, die Diagramme entsprechend zu ergänzen. ;-)
Ich würde das Ergebnis dann auch wieder allen zur Verfügung stellen!

**Comment by kaiherzig** [comment]
Hab dir grade den 2. Update Request geschickt.
Sorry, im ersten war nicht alles gemacht ...
Ich habs als Docker gebuilded. und es tut. Docker updaten musst du noch :)

**Reply by xgcssch** [reply]
Hallo Kai.
Blöde Frage: wo hast du mir ein Update geschickt?
Sönke

**Reply by kaiherzig** [reply]
Hi Sönke,
Ja 2 via GitHub
Ich hab dein Projekt geforked, da es sonst nicht anders ging.

Ich hab die Werte angepasst. 
Der Prometheus exporter war nicht mit dem 2. hk 
Und die Variablen von hk2 waren identisch mit denen von hk1.

Ich hab’s als 1.0.3 getagged.

https://github.com/kaiherzig/SystaComfortPrometheusExporter

Es steht da, dass du informiert wurdest über die Updates.

Ich hab nen docker draus gebaut und es läuft bei mir jetzt seit einer Woche stabil.

Viele Grüße 
Kai

**Reply by xgcssch** [reply]
Ok, dann verstehe ich, was du gemacht hast.
Ich erfahre von Änderungen an deinem Fork nichts. 
Du musst jetzt einen "Pull Request" erstellen. Das ist quasi der Wunsch deine Änderungen in meinen Zweig zurückzuführen und zu übernehmen.
Wenn dann der PR eingearbeitet ist (Erst dann!), kannst du deinen Fork wieder löschen.

**Reply by kaiherzig** [reply]
Ich hab dir 2 push requests geschickt.
Ich seh hier dass es auf Freigabe wartet.

**Reply by kaiherzig** [reply]
Jetzt hast nochmal einen,
Und ich meinte pull requests... nicht Push...
